### ISSUE TITLE
nginx_upstream_check_module模块，标记某个server的状态为down后，如何恢复为up？

### ISSUE THREAD

**Issue by hejinenPrivate** [alibaba/tengine#878]
hi，请教个问题。我看文档使用nginx_upstream_check_module模块进行健康检查，如果某个server连续几次检测连接失败则将其标记为down。请问标记为down后是否还会对这个server进行健康检测，这台server恢复正常以后，tengine会将其标记为up么？

**Comment by djhuahao** [comment]
*   *fall*(fall_count): After fall_count check failures, the server is
        marked down.

   *   *rise*(rise_count): After rise_count check success, the server is
        marked up.

**Comment by hejinenPrivate** [comment]
marked down以后，还会对这个server进行检测吧？如果检测成功数大于rise_count，就会把这个server重新标记为 up，对么？

**Comment by djhuahao** [comment]
是的。去这里详细阅读下说明吧。https://github.com/yaoweibin/nginx_upstream_check_module

**Comment by hejinenPrivate** [comment]
好的 谢谢
